### PR TITLE
PLAT-10866 PLAT-10869

### PR DESCRIPTION
**Pull request by symphony-elias**
### Ticket
PLAT-10866
PLAT-10869

### Description
Replaced native xml library by defusedxml
Improved `if` control structure

### Checklist
- [x] Referenced a ticket in the PR title and in the corresponding section
- [x] Filled properly the description and dependencies, if any
- [N/A] Unit tests updated or added
- [N/A] Docstrings added or updated
- [N/A] Updated the documentation in [docs folder](../docs)

### CODE REVIEW

File: templates/model_utils.mustache [hunk 1]
```diff
@@ -0,0 +1,1602 @@
+{{>partial_header}}
```
Comment (symphony-soufiane) [inline templates/model_utils.mustache:1]: Why did you add the whole file ? was it missing before this change?

**Review by symphony-soufiane — APPROVED** [review]

File: templates/model_utils.mustache (hunk 1 above)
Comment (symphony-elias) [reply templates/model_utils.mustache:1]: Yes it was missing. We didn't have to add it before because we didn't have to change the template.

**Review by symphony-mariacristina — APPROVED** [review]
LGTM!

File: templates/model_utils.mustache (hunk 1 above)
Comment (symphony-youri) [reply templates/model_utils.mustache:1]: and what have we changed in the template?

**Review by symphony-youri — APPROVED** [review]
